project/build.scala

```diff
@@ -289,10 +289,10 @@ object ScalatraBuild extends Build {
     lazy val akkaActor                =  "com.typesafe.akka"       %% "akka-actor"                 % akkaVersion
     lazy val akkaTestkit              =  "com.typesafe.akka"       %% "akka-testkit"               % akkaVersion
     // TODO: 2.3 has incompatible API changes
```

Comment:
Thanks for your contribution. this TODO comment can be removed now (I'll do that later) 😸